features/support/env.rb

```diff
@@ -0,0 +1,31 @@
+require_relative "./../lib/symfony"
+require_relative "./../lib/utils"
+
+PROJECT_ROOT = File.realpath("#{__dir__}/../../")
+FIXTURE_PATH = File.realpath("#{PROJECT_ROOT}/features/fixtures/#{Symfony.fixture}")
+
+# TODO: Maze.hooks.after_configuration doesn't seem to work ?
```

Comment:
Yes, you're right, already noted in PLAT-5666.  I don't think it would be hard to fix if you found you needed it.

Reply:
I don't think it makes a difference for this really